Applications/Spire/Source/TimeAndSales/TimeAndSalesTableView.cpp

```diff
@@ -141,6 +138,17 @@ void TimeAndSalesTableView::resizeEvent(QResizeEvent* event) {
   m_header->setFixedWidth(widget()->width());
 }
 
+void TimeAndSalesTableView::showEvent(QShowEvent* event) {
+  if(!m_was_shown && m_model != nullptr) {
+    m_was_shown = true;
+    m_header->resizeSection(0, scale_width(74));
```

Comment:
Same principle here, this would benefit a lot more from names instead of magic values.